locale/en/blog/weekly-updates/weekly-update.2016-04-23.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Weekly Update - Apr 23rd, 2016
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-04-23T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-04-23
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v5.11.0 (Stable) is released.
+
+### Node v5.11.0 (Stable) Release
+
+We have one release: [Node v5.11.0 (Stable)](https://nodejs.org/en/blog/release/v5.11.0/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```

Comment:
Maybe remove `We have one release:`? Probably no need to mention the number of releases unless there is more than one.